web/src/pages/AppointmentPage.jsx

```diff
@@ -1,21 +1,62 @@
 import { Link } from "react-router-dom";
 import { AppointmentTable } from "../components/AppointmentTable";
-function createData(date, customer, car, service) {
-  return { date, customer, car, service };
-}
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+// function createData(date, customer, car, service) {
```

Comment:
please remove comments

Reply:
Removed comments and also committed this change. Do I need to create another pull request for this?